pkg/nsx/services/common/builder.go

```diff
@@ -65,3 +68,17 @@ func NSXProjectPathToId(path string) (string, string, error) {
 	}
 	return parts[2], parts[4], nil
 }
+
+func IsDefaultNetworkConfigCR(vpcConfigCR *v1alpha1.VPCNetworkConfiguration) bool {
```

Comment:
Similarly we have another isDefaultNetworkConfigCR defined in vpc service with the same content, maybe we can remove that.

Reply:
fixed